lib-object-oriented/line.js

```diff
@@ -1,34 +1,77 @@
 import Tag from './tag'
 
 export default class Line {
-  constructor( content ) {
+  constructor({ content, type }) {
     this.content = content
+    this.tagType = type
+    this.symbolCount = 0
+    this.getSymbolCount()
+    this.newLinesToSpaces()
+  }
+
+  newLinesToSpaces() {
+    const tagType = this.tagType,
```

Comment:
To facilitate maintenance of this code, I would either use the `const` keyword before each declaration on a separate line,  or use a consistently sized tab indent (i.e. 2 spaces).  Otherwise, developers coming after you to work on this code will be forced to multi-space to properly align mew declarations in this function.  Not a deal breaker, but certainly an irritation.

As well, when we have "private" variables like `this.tagType`, `this.content`, and `this.symbolCount`, there is no reason to re-assign them to local variables within the scope of a class' function.